iiif_prezi3/helpers/add_label.py

```diff
@@ -0,0 +1,31 @@
+from ..config import configs
+from ..loader import monkeypatch_schema
+from ..skeleton import (Annotation, AnnotationPage, Canvas, Collection,
+                        Manifest, Range, ResourceItem)
+
+
+class AddLabel:
+
+    def add_label(self, value, language=None):
+        if not self.label:
+            if not language:
+                self.label = value
```

Comment:
Should this set a default lang?

Reply:
I think that might be handled by the AutoLang

Reply:
Yes, handled by AutoLang

Reply:
Don't worry I can see this is set in the auto lang code.